department-of-veterans-affairs/abd-vro
# Add additional validation for the jwt

## What was the problem?
Additional validation for Scope and removal of jwt claims in the header

Associated tickets or Slack threads:
- [MCP-2039](https://amida.atlassian.net/browse/MCP-2039)

## How does this fix it?[^1]
Added validation for the scp

## How to test this PR
- Step 1 : with the jwt token in the secrets project
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
